mmcv/runner/hooks/momentum_updater.py

```diff
@@ -28,8 +30,9 @@ def __init__(self,
         self.warmup_iters = warmup_iters
         self.warmup_ratio = warmup_ratio
 
-        self.base_momentum = []  # initial momentum for all param groups
-        self.regular_momentum = [
+        self.base_momentum: Union[List[float], Dict[str, List[float]]] = [
+        ]  # initial momentum for all param groups
+        self.regular_momentum: Union[List[float], Dict[str, List[float]]] = [
         ]  # expected momentum if no warming up is performed
```

Comment:
The type is a bit long. Can we simplify it with `Union[List, Dict]`?

Reply:
in that case I think it would be better to use `Union[list, dict]`, I think that would be better, will fix this